cypress/integration/auth_spec.js

```diff
@@ -62,7 +62,7 @@ describe('Dashboard authentication', () => {
     });
 
     cy.window().its('appStore').then((store) => {
-      const expirationTime = (new Date(Date.now() - 24 * 3600 * 1000)).valueOf();
+      const expirationTime = (new Date(Date.now() - 24 * 3600 * 1000)).valueOf() / 1000.0;
```

Comment:
Just curious why we're dividing by 1000.0 here

Reply:
the old api/lib/token.js took the input expirationTime and converted it to seconds from ms in the code https://github.com/nasa/cumulus/blob/v1.22.1/packages/api/lib/token.js#L5
The latest doesn't do this anymore.
https://github.com/nasa/cumulus/blob/master/packages/api/lib/token.js#L3-L5